.github/workflows/wasm.yml

```diff
@@ -1,43 +1,51 @@
-name: WASM Build
+name: WebAssembly Build
```

Comment:
I see this replaces the old action.

Reply:
Yeah, as mentioned in the PR message. The old one didn't do much useful - you couldn't use the artifact without having the same version of the compiler used to build it.